libs/platform/src/lib/components/menu-button/menu-button.component.html

```diff
@@ -8,8 +8,9 @@
     [disabled]="disabled"
     (click)="onButtonClick($event)"
     [fdType]="type"
+    tabindex="0"
+    [attr.aria-label]="title"
```

Comment:
As pointed out by Stephan, in our meeting last week, we should only be using `aria-label` in this case if `ng-content` is an empty string. We should programmatically check to see if the text of `ng-content` is zero, and only then assign `aria-label`.

Reply:
will it not be better approach if we set get the [ariaLabel] input and set aria-label. using javascript to just set aria-label does not feel right.

Reply:
The title attribute is not read aloud by some screen readers.  example: JAWS